bundles/org.openhab.core.karaf/src/main/java/org/openhab/core/karaf/internal/FeatureInstaller.java

```diff
@@ -446,24 +478,26 @@ private static synchronized boolean uninstallFeature(FeaturesService featuresSer
         return false;
     }
 
-    private static synchronized boolean installPackage(FeaturesService featuresService,
-            final Map<String, Object> config) {
+    private synchronized boolean installPackage(FeaturesService featuresService, final Map<String, Object> config) {
         boolean configChanged = false;
         Object packageName = config.get(OpenHAB.CFG_PACKAGE);
         if (packageName instanceof String) {
             currentPackage = (String) packageName;
-            String name = PREFIX + PREFIX_PACKAGE + ((String) packageName).trim();
-            if (installFeature(featuresService, name)) {
-                configChanged = true;
+            String fullName = PREFIX + PREFIX_PACKAGE + ((String) packageName).trim();
+            if (currentPackage.equals(MINIMAL_PACKAGE)) {
+                // no changes are done to the add-ons list, so the installer should proceed
+                configChanged = false;
+            } else {
+                if (installFeature(featuresService, fullName)) {
+                    configChanged = true;
+                }
```

Comment:
Should we log anything here or is that already done inside `installFeature` in the case of failure.

Reply:
that's done in installFeature.